docs/whatsnew/6.1.rst

```diff
@@ -21,7 +21,11 @@ Updates to minimum dependencies
 
 The minimum required versions of the following packages have been updated:
 
+- astropy>=6.0.0
 - matplotlib>=3.6.0
+- numpy>=1.24.0
+- pandas>=1.5.0
+- sphinx>=6.0.0
```

Comment:
I am not sure that this matters to anyone?

Reply:
Every once in a while someone builds the docs locally...